### ISSUE TITLE
Suggestion - Local Contracts

### ISSUE THREAD

**Issue by CloudTheWolf** [InZidiuZ/op-framework-issue-tracker#3145]
### Assurance

I have read the `README.md`

### Summary

To add more RP and legal opportunities to the city, being able to take Local Contracts from NPC businesses (Eg 24/7, Cluckin' Bell, Post Op, Steel Mill, Lumber Yard, etc)  

Examples of possible jobs:

 - 24/7: "A customer on Grove Street has placed a delivery order" (Player gets a bag with all the items and a marker on their map, maybe a time limit event where the reward is lower)
 
 - Cluckin' Bell: a few different options, but an example could be the player goes to the Cluckin' bell on Rockford Plaza and has to deliver food (Think Uber Eats) or they are asked to go to the Cluckin bell factory to get some more stock as they are running out of wings. 

- Post Op: Basically think Amazon Deliveries, Get so packages in your car, then deliver them (Payout depends on time take, and damage to goods)

- Steel Mill / Lumber yard: Player must be in a van and is asked to deliver materials to one of the construction sites around the map (Not just in the city, but maybe somewhere in sandy and the construction site in Paleto)

There should be a limit on how many you can do either in a day or maybe every 2 hours.

Additionally, there could be some more "Illegal" ones available that could reward players with useful information in the form of hints.

Eg once you do the job, the NPC says "I hear you can make an {Item Name}." followed by either a general area 
Eg "You just need to take the items somewhere the area of Vinewood Hills"

Or a random item needed to craft the items:

Eg:

"I think you need a CPU as well as some other items, I can't remember"

This could also be added as a Daily Activity as the bazar

"Do X contract job(s)" 

### Reason

This would be a good way for Legal players to explore the city looking for more to do, As well as for legal players as there would now be a unique way to learn about new craftables fully in character.

**Comment by HollaYalla** [comment]
This would be awesome to have for people with WL jobs to make some extra cash

**Comment by coalaura** [comment]
> Post Op

Thats just the delivery job, no?

**Comment by HollaYalla** [comment]
> Thats just the delivery job, no?

Yes but I think he means a different variant that isn’t a job or I guess a more part time / restricted version

**Comment by CloudTheWolf** [comment]
> > Post Op
> 
> Thats just the delivery job, no?

That's what my initial thought for this one is based on, but then add a difficulty curve to it in that you start at Post Op in Palito, have a time limit, and could damage the goods being delivered which could impact the overall reward.

**Comment by DopeIndar** [comment]
It is just similar to Post office job what you suggested but the Idea of a local contract is good indeed. The Cluckin Bell Uber Eats Idea Is Better. Like Pxy you geta Contract Player will get a Contract from Cluckin Bell to collect and bring them their Stuff basically it's a like a Oxy job but for White collar people.
